### PR TITLE
made metastatus read from /etc/paasta to get dashboards

### PR DESCRIPTION
**Pull request by mjksmith**
Sample output:
```
matts@dev2-devc:~/paasta (master) $ python paasta_tools/cli/cli.py metastatus -c nova-prod
Cluster: nova-prod
Note:              Viewing raw dashboards in prod requires an extra proxy. See https://docs.google.com/document/d/1OXEcmSubOG8T0RHFETh3GM98AF0MXj-1vQL6x8xLJO0/edit#heading=h.anrifrwk9e2r
Chronos RO:        http://chronos-nova-prod.yelpcorp.com (LDAP Login)
Mesos:             http://mesos.paasta-nova-prod.yelp (Requires Prod Proxy, see note at the end)
To run metastatus: paasta metastatus -c nova-prod (requires prod access)
Smartstack:        http://paasta-nova-prod.yelp:3212  (Requires Prod Proxy, see note at the end)
Signalfx:          https://app.signalfx.com/#/dashboard/CYJklh9AYAo?sources%5B%5D=superregion:nova-prod
Marathon RO:       http://marathon-nova-prod.yelpcorp.com (LDAP Login)
Mesos Status: OK
Marathon Status: OK
Chronos Status: OK
```

### CODE REVIEW

**Comment by mjksmith** [comment]
AFAICT the only way we could preserve the ordering from `/etc/paasta` would be to make `/etc/paasta` use a `list` instead of a `dict` or to make `SystemPaastaConfig` a `dict` of `OrderedDict`s (editing `load_system_paasta_config`)

File: paasta_tools/utils.py [hunk 1]
```diff
@@ -633,6 +633,13 @@ def get_fsm_cluster_map(self):
             raise PaastaNotConfiguredError(
                 'Could not find fsm_cluster_map in configuration directory: %s' % self.directory)
 
+    def get_dashboard_links(self):
+        try:
+            return self['dashboard_links']
```
Comment (solarkennedy) [inline paasta_tools/utils.py]: Don't crash so hard on this. It is ok to just return no links if we don't have any set yet.

**Comment by solarkennedy** [comment]
Can you maybe indent a bit to make it more obvious what this text is?

```
Dashboards:
  foo: bar
Mesos Status:
Marathon Status:
...
```

**Comment by solarkennedy** [comment]
fix + ship

**Pull request merged**